firebase/headers.js

```diff
@@ -65,52 +65,33 @@ module.exports = [
             'status.maxmind.com',
             'www.google-analytics.com',
             'www.googleapis.com',
-            'https://ekr.zdassets.com',
-            'https://maxmind.zendesk.com',
-            'https://static.zdassets.com',
-            'https://*.zopim.com',
-            'wss://maxmind.zendesk.com',
-            'wss://*.zopim.com',
             'https://www.maxmind.com',
+
+            // HubSpot chat
+            'https://api.hubspot.com',
+            'https://forms.hscollectedforms.net',
+            'https://forms.hsforms.com',
```

Comment:
I'm not sure what all we technically need here, but https://knowledge.hubspot.com/domains-and-urls/ssl-and-domain-security-in-hubspot#content-security-policy lists both `hsforms.com` and `hsforms.net`